### DISCUSSION TITLE
How to run nvim with the minimal config possible

### DISCUSSION THREAD

**Discussion by 2ax3ax5** [neovim/neovim#25308] [Q&A]
I've got two questions related to run nvim as agnostic and about everything as possible:

A) Is there a way to run nvim without loading anything at all. If I run `nvim --noplugin --clean` from a fresh installation from this repository, `scriptnames` show 5 scripts `ftplugin, indent, filetype, syntax/syntax, syntax/synload`. What would be the best way to do so? 

B) What is the best way to undefine (or set to nothing) all the bindings that can be undefined?

**Comment by zeertzjq** [comment]
A) `nvim -u NONE -i NONE`
B) Depends on what you mean by "bindings". `:mapclear | :mapclear!` removes all mappings.

**Reply by 2ax3ax5** [reply]
A) Perfect.

B) To any map, binding, shortcut, and alike. For instance, in stty I don't have maped ctrl+z, but it seems vim add it. I can remap <c-z> for every mode and so, but I was wondering if it was a way to undefine all these maps in a line or two.

Thank you.
